src/toast.ts

```diff
@@ -57,6 +57,12 @@ export default class GlideCoreToast extends LitElement {
   @property({ type: Boolean })
   privateDismissingViaButton = false;
 
+  // `privateId` instead of `id` so consumers can specify their own `id` to determine,
+  // via `event.target.id`, which Toast was dismissed and then update whatever data
+  // structure they're using to render Toasts.
+  @property()
+  privateId = nanoid();
```

Comment:
@ynotdraw: Changed this to `privateId` based on your offline feedback. As a bonus, the story is [slightly simpler](https://github.com/CrowdStrike/glide-core/pull/847/commits/a9bf7d850df70d34ef48e0aee88fc5572b3c18a8#diff-e0509877cd4a4c5dedd559866f580a1064c09704cceda178618dc4bd327a75f1).

Reply:
Sweet, looks good to me! Thanks